pom.xml

```diff
@@ -79,6 +79,8 @@
               <buildArgs>
                 <buildArg>--no-fallback</buildArg>
                 <buildArg>--no-server</buildArg>
+                <buildArg>--initialize-at-build-time=com.example</buildArg>
```

Comment:
The background of this option was that, when I tried 

```
  static {
    String myProp = System.getProperty("my.prop");
    System.out.println("myProp is " + myProp);
  }
```

, then I found that this class is not initialized at build time.

Reply:
This is awesome! An extremely neat solution!

In previous attempts, I created a separate class that introduced `static final String myProp= System.getProperty("my.prop");` and initialized that at build time in hopes that the static variable would be accessible later on. But I was wrong in my understanding of this because the application itself was being executed at run time and hence didn't have access to the properties initialized at build time.

Reply:
> @mpeddada1 This works. Do you think it's applicable to java-spanner repository?

Tracking some attempts to follow this pattern here: https://github.com/mpeddada1/java-spanner/pull/1